antarest/study/web/studies_blueprint.py

```diff
@@ -815,6 +816,26 @@ def unarchive_output(
         )
         return content
 
+    @bp.get(
+        "/private/studies/{study_id}/outputs/{output_id}/digest",
+        tags=[APITag.study_outputs],
+        summary="Get an output digest file",
+        response_model=DigestDTO,
```

Comment:
I don't think this is needed ?

Reply:
You talk about the response model ? If so indeed i think it's not needed but why would I remove it ?